items/de.json

```
@@ -0,0 +1,202 @@
+{
+  "42": "42",
+  "air": "Luft",
+  "alchemistry": "Alchemie",
+  "algae": "Algen",
+  "alien": "Alien",
+  "animal": "Tier",
+  "aquarium": "Aquarium",
+  "archipelago": "Archipel",
+  "ash": "Asche",
+  "astronaut": "Astronaut",
+  "atoll": "Atoll",
+  "axe": "Axt",
+  "baby": "Baby",
+  "beach": "Strand",
+  "beach_castle": "Sandburg",
+  "beverage": "Trinken",
+  "binary_system": "Binäres System",
+  "bird": "Vogel",
+  "black_hole": "Schwarzes Loch",
+  "blood": "Blut",
+  "boat": "Boot",
+  "bone": "Knochen",
+  "book": "Buch",
+  "box": "Kiste",
+  "bread": "Brot",
+  "breaded_fish": "Fischbrötchen",
```

```suggestion
  "breaded_fish": "Fischstäbchen",
```
I think it's this one instead, I let you confirm